src/SWP/Bundle/TemplatesSystemBundle/Resources/config/doctrine/Container.orm.yml

```diff
@@ -42,15 +42,15 @@ SWP\Bundle\TemplateEngineBundle\Model\Container:
             gedmo:
                 timestampable:
                     on: update
-        tenantCode:
-            type: string
-    uniqueConstraints:
-        swp_name_idx:
-          columns: [ name, tenant_code ]
+#        tenantCode:
```

Comment:
so currently containers are not tenant aware? If so this `Container` class should be extended by Container class in Core Bundle and the `tenantCode` property should be added and the new class should be used by TemplatesSystemBundle instead